model/task/db.go

```diff
@@ -138,7 +138,14 @@ var (
 		},
 	},
 	}
-
+	isAttainable = bson.M{
```

Comment:
could you drop a comment explaining what this is doing, just for future reference?